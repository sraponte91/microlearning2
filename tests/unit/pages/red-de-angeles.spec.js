import { config, createLocalVue, mount } from '@vue/test-utils'
import VueMeta from 'vue-meta'
import RedDeAngeles from '@/pages/red-de-angeles.vue'
// create vue with vue-meta
const localVue = createLocalVue()
localVue.use(VueMeta, { keyName: 'head' })

const mock = {
    localVue,
    mocks: {}
}

const wrapper = mount(RedDeAngeles, mock)

describe('red-de-angeles.vue', () => {
    test('Is a Vue instance', () => {
        expect(wrapper.vm).toBeTruthy()
    })
    test('Renders correctly', () => {
        expect(wrapper.element).toMatchSnapshot();
    });
    test('has correct <head> content', () => {
        expect(wrapper.vm.$metaInfo.title).toBe(config.mocks['$constants'].redDeAngeles.meta.title + ' - ' +
            config.mocks['$constants'].home.meta.title)

        expect(wrapper.vm.$metaInfo.meta.find(
            (item) => item.hid === 'title'
        ).content).toBe(config.mocks['$constants'].redDeAngeles.meta.metaTitle + ' - ' +
            config.mocks['$constants'].home.meta.title)

        expect(wrapper.vm.$metaInfo.meta.find(
            (item) => item.hid === 'description'
        ).content).toBe(config.mocks['$constants'].redDeAngeles.meta.metaDescription)

        expect(wrapper.vm.$metaInfo.meta.find(
            (item) => item.hid === 'og:title'
        ).content).toBe(config.mocks['$constants'].redDeAngeles.meta.ogTitle + ' - ' +
            config.mocks['$constants'].home.meta.title)

        expect(wrapper.vm.$metaInfo.meta.find(
            (item) => item.hid === 'og:description'
        ).content).toBe(config.mocks['$constants'].redDeAngeles.meta.ogDescription)

        expect(wrapper.vm.$metaInfo.meta.find(
            (item) => item.hid === 'og:image'
        ).content).toBe(config.mocks['$constants'].baseURL + config.mocks['$constants'].redDeAngeles.meta.ogImage)
    })
})
