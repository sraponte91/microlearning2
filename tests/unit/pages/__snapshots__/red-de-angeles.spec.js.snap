// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`red-de-angeles.vue Renders correctly 1`] = `
<main>
  <section
    class="invester-section-1"
  >
    <div
      class="container margin-0-auto"
    >
      <div
        class="row"
      >
        <div
          class="col-md-12 text-center leader-title"
        >
          <h1
            class="ff-sans-b"
          >
            Red de Ángeles inversionistas
          </h1>
           
          <p
            class="text-orange"
          >
            
            Un espacio donde su liderazgo tendrá impacto nacional
          
          </p>
        </div>
         
        <div
          class="col-md-12"
        >
          <div
            class="sec1-boxes"
          >
            <img
              alt=""
              class="img-fluid"
              src="/images/invester-about.jpg"
            />
             
            <div
              class="sec1-description"
            >
              <h3
                class="ff-sans-b"
              >
                ¿Quiénes Somos?
              </h3>
               
              <p>
                
                La Red de Ángeles Inversionistas es un programa que apuesta
                por la transformación social de negocios rentables y
                sostenibles. Esta Red busca generar un cambio en la narrativa
                de la inversión de riesgo y de impacto para crecer como
                sociedad. Así mismo, creamos espacios y conexiones de valor
                para apoyar a los empresarios locales por medio de
                herramientas y metodologías que les permitan fortalecer sus
                capacidades para el levantamiento de capital e inversión con
                impacto.
              
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>
   
  <section
    class="invester-section-2 bg-grey"
  >
    <div
      class="container margin-0-auto"
    >
      <h2
        class="ff-sans-b"
      >
        Perfiles
      </h2>
       
      <div
        class="row"
      >
        <div
          class="col-md-12"
        >
          <div
            class="sec2-boxes"
          >
            <div
              class="sec2-inner-box"
            >
              <h4
                class="ff-sans-b"
              >
                Empresas
              </h4>
               
              <img
                alt=""
                class="img-fluid"
                src="/images/invester-profile-1.jpg"
              />
               
              <p>
                
                Una vez cursados los seis módulos del plan de estudios, los
                miembros de la cohorte aplican el conocimiento y las
                habilidades adquiridas, para escribir y presentar un Reto de
                Emprendimiento. Esta evaluación final puede ser muy versátil,
                dependiendo de los intereses, creatividad y antecedentes de
                cada participante.
              
              </p>
               
              <div
                class="text-center"
              >
                <a
                  class="btn btn-primary"
                  href="https://fundacionbolivardaviviendaorg.force.com/comunidadfbd/s/?language=es_CO&programa=Red+de+angeles+inversionistas&grupo=Empresas+Red+de+Angeles"
                  rel="noopener"
                  target="_blank"
                >
                  Inscribase aquí
                </a>
              </div>
            </div>
             
            <div
              class="sec2-inner-box"
            >
              <h4
                class="ff-sans-b"
              >
                Inversionista
              </h4>
               
              <img
                alt=""
                class="img-fluid"
                src="/images/invester-profile-2.jpg"
              />
               
              <p>
                
                Los inversionistas que participan en la Red de Ángeles
                Inversionistas han impactado al país con su liderazgo y
                extensa experiencia dentro de su industria. Son, además,
                profesionales que cuentan con un espíritu emprendedor innato,
                y tienen una capacidad de inversión entre el 5% y el 20% de su
                patrimonio, con la cual le apostarán a start-ups o en empresas
                en una etapa temprana.
              
              </p>
               
              <div
                class="text-center"
              >
                <a
                  class="btn btn-primary"
                  href="https://fundacionbolivardaviviendaorg.force.com/comunidadfbd/s/?language=es_CO&programa=Red+de+angeles+inversionistas&grupo=Inversionistas"
                  rel="noopener"
                  target="_blank"
                >
                  Inscribase aquí
                </a>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>
   
  <section
    class="invester-section-3"
  >
    <div
      class="container margin-0-auto"
    >
      <div
        class="row"
      >
        <div
          class="col-md-8 offset-md-2"
        >
          <h2
            class="ff-sans-b"
          >
            
            Requisitos para aplicar a la red como emprendendor
          
          </h2>
           
          <div>
            <div
              class="VueCarousel"
            >
              <div
                class="VueCarousel-wrapper"
              >
                <div
                  class="VueCarousel-inner"
                  style="transform: translate(0px, 0); transition: 0.5s ease transform; flex-basis: 0px; visibility: hidden; height: auto;"
                >
                  <div
                    aria-hidden="true"
                    class="VueCarousel-slide"
                    role="tabpanel"
                    tabindex="-1"
                  >
                    <div
                      class="sec3-boxes"
                    >
                      <div
                        class="row align-items-center"
                      >
                        <div
                          class="col-md-4"
                        >
                          <img
                            alt=""
                            class="img-fluid"
                            src="/images/icon-user-group.png"
                          />
                        </div>
                         
                        <div
                          class="col-md-8"
                        >
                          <p>
                            
                        Su emprendimiento no destruye valor social o ambiental
                      
                          </p>
                        </div>
                      </div>
                    </div>
                  </div>
                  <div
                    aria-hidden="true"
                    class="VueCarousel-slide"
                    role="tabpanel"
                    tabindex="-1"
                  >
                    <div
                      class="sec3-boxes"
                    >
                      <div
                        class="row align-items-center"
                      >
                        <div
                          class="col-md-4"
                        >
                          <img
                            alt=""
                            class="img-fluid"
                            src="/images/icon-user-group.png"
                          />
                        </div>
                         
                        <div
                          class="col-md-8"
                        >
                          <p>
                            
                        Su emprendimiento no destruye valor social o ambiental
                      
                          </p>
                        </div>
                      </div>
                    </div>
                  </div>
                  <div
                    aria-hidden="true"
                    class="VueCarousel-slide"
                    role="tabpanel"
                    tabindex="-1"
                  >
                    <div
                      class="sec3-boxes"
                    >
                      <div
                        class="row align-items-center"
                      >
                        <div
                          class="col-md-4"
                        >
                          <img
                            alt=""
                            class="img-fluid"
                            src="/images/icon-user-group.png"
                          />
                        </div>
                         
                        <div
                          class="col-md-8"
                        >
                          <p>
                            
                        Su emprendimiento no destruye valor social o ambiental
                      
                          </p>
                        </div>
                      </div>
                    </div>
                  </div>
                  <div
                    aria-hidden="true"
                    class="VueCarousel-slide"
                    role="tabpanel"
                    tabindex="-1"
                  >
                    <div
                      class="sec3-boxes"
                    >
                      <div
                        class="row align-items-center"
                      >
                        <div
                          class="col-md-4"
                        >
                          <img
                            alt=""
                            class="img-fluid"
                            src="/images/icon-user-group.png"
                          />
                        </div>
                         
                        <div
                          class="col-md-8"
                        >
                          <p>
                            
                        Su emprendimiento no destruye valor social o ambiental
                      
                          </p>
                        </div>
                      </div>
                    </div>
                  </div>
                  <div
                    aria-hidden="true"
                    class="VueCarousel-slide"
                    role="tabpanel"
                    tabindex="-1"
                  >
                    <div
                      class="sec3-boxes"
                    >
                      <div
                        class="row align-items-center"
                      >
                        <div
                          class="col-md-4"
                        >
                          <img
                            alt=""
                            class="img-fluid"
                            src="/images/icon-user-group.png"
                          />
                        </div>
                         
                        <div
                          class="col-md-8"
                        >
                          <p>
                            
                        Su emprendimiento no destruye valor social o ambiental
                      
                          </p>
                        </div>
                      </div>
                    </div>
                  </div>
                  <div
                    aria-hidden="true"
                    class="VueCarousel-slide"
                    role="tabpanel"
                    tabindex="-1"
                  >
                    <div
                      class="sec3-boxes"
                    >
                      <div
                        class="row align-items-center"
                      >
                        <div
                          class="col-md-4"
                        >
                          <img
                            alt=""
                            class="img-fluid"
                            src="/images/icon-user-group.png"
                          />
                        </div>
                         
                        <div
                          class="col-md-8"
                        >
                          <p>
                            
                        Su emprendimiento no destruye valor social o ambiental
                      
                          </p>
                        </div>
                      </div>
                    </div>
                  </div>
                  <div
                    class="VueCarousel-slide VueCarousel-slide-active VueCarousel-slide-center"
                    role="tabpanel"
                    tabindex="-1"
                  >
                    <div
                      class="sec3-boxes"
                    >
                      <div
                        class="row align-items-center"
                      >
                        <div
                          class="col-md-4"
                        >
                          <img
                            alt=""
                            class="img-fluid"
                            src="/images/icon-user-group.png"
                          />
                        </div>
                         
                        <div
                          class="col-md-8"
                        >
                          <p>
                            
                        Su emprendimiento no destruye valor social o ambiental
                      
                          </p>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
               
              <!---->
               
              <div
                class="VueCarousel-pagination"
                data-v-438fd353=""
                style=""
              >
                <div
                  class="VueCarousel-dot-container"
                  data-v-438fd353=""
                  role="tablist"
                  style="margin-top: 20px;"
                >
                  <button
                    aria-hidden="false"
                    aria-label="Item 0"
                    aria-selected="false"
                    class="VueCarousel-dot"
                    data-v-438fd353=""
                    role="tab"
                    style="margin-top: 20px; padding: 10px; width: 10px; height: 10px; background-color: rgb(239, 239, 239);"
                    title="Item 0"
                    value="Item 0"
                  />
                  <button
                    aria-hidden="false"
                    aria-label="Item 1"
                    aria-selected="false"
                    class="VueCarousel-dot"
                    data-v-438fd353=""
                    role="tab"
                    style="margin-top: 20px; padding: 10px; width: 10px; height: 10px; background-color: rgb(239, 239, 239);"
                    title="Item 1"
                    value="Item 1"
                  />
                  <button
                    aria-hidden="false"
                    aria-label="Item 2"
                    aria-selected="false"
                    class="VueCarousel-dot"
                    data-v-438fd353=""
                    role="tab"
                    style="margin-top: 20px; padding: 10px; width: 10px; height: 10px; background-color: rgb(239, 239, 239);"
                    title="Item 2"
                    value="Item 2"
                  />
                  <button
                    aria-hidden="false"
                    aria-label="Item 3"
                    aria-selected="false"
                    class="VueCarousel-dot"
                    data-v-438fd353=""
                    role="tab"
                    style="margin-top: 20px; padding: 10px; width: 10px; height: 10px; background-color: rgb(239, 239, 239);"
                    title="Item 3"
                    value="Item 3"
                  />
                  <button
                    aria-hidden="false"
                    aria-label="Item 4"
                    aria-selected="false"
                    class="VueCarousel-dot"
                    data-v-438fd353=""
                    role="tab"
                    style="margin-top: 20px; padding: 10px; width: 10px; height: 10px; background-color: rgb(239, 239, 239);"
                    title="Item 4"
                    value="Item 4"
                  />
                  <button
                    aria-hidden="false"
                    aria-label="Item 5"
                    aria-selected="false"
                    class="VueCarousel-dot"
                    data-v-438fd353=""
                    role="tab"
                    style="margin-top: 20px; padding: 10px; width: 10px; height: 10px; background-color: rgb(239, 239, 239);"
                    title="Item 5"
                    value="Item 5"
                  />
                  <button
                    aria-hidden="false"
                    aria-label="Item 6"
                    aria-selected="true"
                    class="VueCarousel-dot VueCarousel-dot--active"
                    data-v-438fd353=""
                    role="tab"
                    style="margin-top: 20px; padding: 10px; width: 10px; height: 10px; background-color: rgb(0, 0, 0);"
                    title="Item 6"
                    value="Item 6"
                  />
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>
   
  <section
    class="invester-section-4 bg-grey testimonios-sec"
  >
    <div
      class="container-fluid"
    >
      <div
        class="sec-title text-center"
      >
        <h2
          class="ff-sans-b"
        >
          Testimonios
        </h2>
      </div>
       
      <div
        class="testimo-slider"
      >
        <div
          class="for-marg-1"
        >
          <div>
            <div
              class="VueCarousel"
            >
              <div
                class="VueCarousel-wrapper"
              >
                <div
                  class="VueCarousel-inner"
                  style="transform: translate(0px, 0); transition: 0.5s ease transform; flex-basis: 0px; visibility: hidden; height: auto;"
                >
                  <div
                    aria-hidden="true"
                    class="VueCarousel-slide"
                    role="tabpanel"
                    tabindex="-1"
                  >
                    <div
                      class="position-relative testimo-bx bg-white"
                    >
                      <div
                        class="text-center til ff-sans-b"
                      >
                        <h3>
                          
                      Carolina Escalona - Presidenta Fundación Rafael Escalona
                    
                        </h3>
                      </div>
                       
                      <div
                        class="txt margin-0-auto position-relative after-po before-po quote-sign ff-sans-i"
                      >
                        <p>
                          
                      Con Aflora pudimos realizar muchos cambios internos y reconocer qué nos hace falta para llegar a ser una organización autosostenible y con visión.
                    
                        </p>
                      </div>
                    </div>
                  </div>
                  <div
                    aria-hidden="true"
                    class="VueCarousel-slide"
                    role="tabpanel"
                    tabindex="-1"
                  >
                    <div
                      class="position-relative testimo-bx bg-white"
                    >
                      <div
                        class="text-center til ff-sans-b"
                      >
                        <h3>
                          
                      Marian Tertel - Coordinadora en la Fundación Sole Colombia
                    
                        </h3>
                      </div>
                       
                      <div
                        class="txt margin-0-auto position-relative after-po before-po quote-sign ff-sans-i"
                      >
                        <p>
                          
                      Gracias a Aflora pudimos entender dónde estábamos y hacia dónde teníamos que proyectarnos para cumplir todos nuestros objetivos.
                    
                        </p>
                      </div>
                    </div>
                  </div>
                  <div
                    aria-hidden="true"
                    class="VueCarousel-slide"
                    role="tabpanel"
                    tabindex="-1"
                  >
                    <div
                      class="position-relative testimo-bx bg-white"
                    >
                      <div
                        class="text-center til ff-sans-b"
                      >
                        <h3>
                          
                      Isabel Cristina Rengifo - Representante legal de Fundación Mis Corazones Alegres
                    
                        </h3>
                      </div>
                       
                      <div
                        class="txt margin-0-auto position-relative after-po before-po quote-sign ff-sans-i"
                      >
                        <p>
                          
                      Con los ciclos de Aflora logramos armar bien los proyectos y fortalecer los  procesos internos de la fundación
                    
                        </p>
                      </div>
                    </div>
                  </div>
                  <div
                    aria-hidden="true"
                    class="VueCarousel-slide"
                    role="tabpanel"
                    tabindex="-1"
                  >
                    <div
                      class="position-relative testimo-bx bg-white"
                    >
                      <div
                        class="text-center til ff-sans-b"
                      >
                        <h3>
                          
                      Isabel Cristina Rengifo - Representante legal de Fundación Mis Corazones Alegres
                    
                        </h3>
                      </div>
                       
                      <div
                        class="txt margin-0-auto position-relative after-po before-po quote-sign ff-sans-i"
                      >
                        <p>
                          
                      Con los ciclos de Aflora logramos armar bien los proyectos y fortalecer los  procesos internos de la fundación
                    
                        </p>
                      </div>
                    </div>
                  </div>
                  <div
                    aria-hidden="true"
                    class="VueCarousel-slide"
                    role="tabpanel"
                    tabindex="-1"
                  >
                    <div
                      class="position-relative testimo-bx bg-white"
                    >
                      <div
                        class="text-center til ff-sans-b"
                      >
                        <h3>
                          
                      Isabel Cristina Rengifo - Representante legal de Fundación Mis Corazones Alegres
                    
                        </h3>
                      </div>
                       
                      <div
                        class="txt margin-0-auto position-relative after-po before-po quote-sign ff-sans-i"
                      >
                        <p>
                          
                      Con los ciclos de Aflora logramos armar bien los proyectos y fortalecer los  procesos internos de la fundación
                    
                        </p>
                      </div>
                    </div>
                  </div>
                  <div
                    aria-hidden="true"
                    class="VueCarousel-slide"
                    role="tabpanel"
                    tabindex="-1"
                  >
                    <div
                      class="position-relative testimo-bx bg-white"
                    >
                      <div
                        class="text-center til ff-sans-b"
                      >
                        <h3>
                          
                      Isabel Cristina Rengifo - Representante legal de Fundación Mis Corazones Alegres
                    
                        </h3>
                      </div>
                       
                      <div
                        class="txt margin-0-auto position-relative after-po before-po quote-sign ff-sans-i"
                      >
                        <p>
                          
                      Con los ciclos de Aflora logramos armar bien los proyectos y fortalecer los  procesos internos de la fundación
                    
                        </p>
                      </div>
                    </div>
                  </div>
                  <div
                    class="VueCarousel-slide VueCarousel-slide-active"
                    role="tabpanel"
                    tabindex="-1"
                  >
                    <div
                      class="position-relative testimo-bx bg-white"
                    >
                      <div
                        class="text-center til ff-sans-b"
                      >
                        <h3>
                          
                      Isabel Cristina Rengifo - Representante legal de Fundación Mis Corazones Alegres
                    
                        </h3>
                      </div>
                       
                      <div
                        class="txt margin-0-auto position-relative after-po before-po quote-sign ff-sans-i"
                      >
                        <p>
                          
                      Con los ciclos de Aflora logramos armar bien los proyectos y fortalecer los  procesos internos de la fundación
                    
                        </p>
                      </div>
                    </div>
                  </div>
                  <div
                    class="VueCarousel-slide VueCarousel-slide-active"
                    role="tabpanel"
                    tabindex="-1"
                  >
                    <div
                      class="position-relative testimo-bx bg-white"
                    >
                      <div
                        class="text-center til ff-sans-b"
                      >
                        <h3>
                          
                      Isabel Cristina Rengifo - Representante legal de Fundación Mis Corazones Alegres
                    
                        </h3>
                      </div>
                       
                      <div
                        class="txt margin-0-auto position-relative after-po before-po quote-sign ff-sans-i"
                      >
                        <p>
                          
                      Con los ciclos de Aflora logramos armar bien los proyectos y fortalecer los  procesos internos de la fundación
                    
                        </p>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
               
              <!---->
               
              <div
                class="VueCarousel-pagination"
                data-v-438fd353=""
                style=""
              >
                <div
                  class="VueCarousel-dot-container"
                  data-v-438fd353=""
                  role="tablist"
                  style="margin-top: 20px;"
                >
                  <button
                    aria-hidden="false"
                    aria-label="Item 0"
                    aria-selected="false"
                    class="VueCarousel-dot"
                    data-v-438fd353=""
                    role="tab"
                    style="margin-top: 20px; padding: 10px; width: 10px; height: 10px; background-color: rgb(239, 239, 239);"
                    title="Item 0"
                    value="Item 0"
                  />
                  <button
                    aria-hidden="false"
                    aria-label="Item 1"
                    aria-selected="false"
                    class="VueCarousel-dot"
                    data-v-438fd353=""
                    role="tab"
                    style="margin-top: 20px; padding: 10px; width: 10px; height: 10px; background-color: rgb(239, 239, 239);"
                    title="Item 1"
                    value="Item 1"
                  />
                  <button
                    aria-hidden="false"
                    aria-label="Item 2"
                    aria-selected="false"
                    class="VueCarousel-dot"
                    data-v-438fd353=""
                    role="tab"
                    style="margin-top: 20px; padding: 10px; width: 10px; height: 10px; background-color: rgb(239, 239, 239);"
                    title="Item 2"
                    value="Item 2"
                  />
                  <button
                    aria-hidden="false"
                    aria-label="Item 3"
                    aria-selected="true"
                    class="VueCarousel-dot VueCarousel-dot--active"
                    data-v-438fd353=""
                    role="tab"
                    style="margin-top: 20px; padding: 10px; width: 10px; height: 10px; background-color: rgb(0, 0, 0);"
                    title="Item 3"
                    value="Item 3"
                  />
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>
   
  <section
    class="invester-section-5"
  >
    <div
      class="container margin-0-auto"
    >
      <h4
        class="ff-sans-b"
      >
        Con el apoyo de:
      </h4>
       
      <div
        class="d-flex flex-wrap for-marg-1"
      >
        <div
          class="lgoes-bx d-flex align-items-center position-relative"
        >
          <img
            alt=""
            src="/images/gell_logo.svg"
          />
           
          <a
            class="position-absolute full-box-link"
            href="javascript:void(0)"
          />
        </div>
         
        <div
          class="lgoes-bx d-flex align-items-center position-relative"
        >
          <img
            alt=""
            src="/images/gell_logo.svg"
          />
           
          <a
            class="position-absolute full-box-link"
            href="javascript:void(0)"
          />
        </div>
         
        <div
          class="lgoes-bx d-flex align-items-center position-relative"
        >
          <img
            alt=""
            src="/images/gell_logo.svg"
          />
           
          <a
            class="position-absolute full-box-link"
            href="javascript:void(0)"
          />
        </div>
         
        <div
          class="lgoes-bx d-flex align-items-center position-relative"
        >
          <img
            alt=""
            src="/images/gell_logo.svg"
          />
           
          <a
            class="position-absolute full-box-link"
            href="javascript:void(0)"
          />
        </div>
         
        <div
          class="lgoes-bx d-flex align-items-center position-relative"
        >
          <img
            alt=""
            src="/images/gell_logo.svg"
          />
           
          <a
            class="position-absolute full-box-link"
            href="javascript:void(0)"
          />
        </div>
         
        <div
          class="lgoes-bx d-flex align-items-center position-relative"
        >
          <img
            alt=""
            src="/images/gell_logo.svg"
          />
           
          <a
            class="position-absolute full-box-link"
            href="javascript:void(0)"
          />
        </div>
         
        <div
          class="lgoes-bx d-flex align-items-center position-relative"
        >
          <img
            alt=""
            src="/images/gell_logo.svg"
          />
           
          <a
            class="position-absolute full-box-link"
            href="javascript:void(0)"
          />
        </div>
      </div>
    </div>
  </section>
</main>
`;
